app/src/main/java/ru/vssemikoz/newsfeed/models/News.java

```diff
@@ -0,0 +1,50 @@
+package ru.vssemikoz.newsfeed.models;
+
+import java.net.URL;
+import java.util.Date;
+
+public class News {
```

Comment:
What do you think about renaming this class to `NewsItem`? It's not news basically but a single item.